tools/azure-sdk-tools/packaging_tools/generate_client.py

```diff
@@ -40,19 +40,24 @@ def generate_autorest(folder: Path) -> None:
 
 
 def generate_typespec(folder: Path) -> None:
+    tsp_location_path = folder / "tsp-location.yaml"
 
-    # Turns out, "pwsh" is the name for PowerShell 7 on Windows, that is required for those scripts
-    ps_cmd = "pwsh"
+    if not tsp_location_path.exists():
+        raise ValueError(
+            "Didn't find a tsp_location.yaml in local directory. Please provide a valid "
```

Comment:
nit: wording. Do we need to actually provide one or does it just need to exist in the directory?

Reply:
It just needs to exist, thanks for pointing that out! Just updated the wording :)